tests/cephfs/cephfs_utilsV1.py

```diff
@@ -831,3 +831,22 @@ def subvolume_authorize(self, client, vol_name, subvol_name, client_name, **kwar
         if kwargs.get("extra_params"):
             command += f" {kwargs.get('extra_params')}"
             client.exec_command(sudo=True, cmd=command)
+
+    def heartbeat_map(self, mds):
+        """
+        Verify "heartbeat_map" timeout is not in mds log
+        Args:
+            mds: mds node
+        """
+        try:
+            mds.exec_command(
+                sudo=True,
+                cmd="cat /var/log/ceph/ceph-mds.%s.log "
+                "| grep heartbeat_map" % mds.node.hostname,
```

Comment:
IMO, please use `mds.node.shortname` instead of `mds.node.hostname`. `hostname` is FQDN